modules/vba_documents/spec/sidekiq/upload_processor_spec.rb

```diff
@@ -191,6 +191,8 @@
       expect(capture_body).to have_key('document')
       metadata = JSON.parse(capture_body['metadata'])
       expect(metadata['uuid']).to eq(upload.guid)
+      # leading\trailing whitespace should have been removed
+      expect(metadata['fileNumber']).to eq(metadata['fileNumber']&.strip)
```

Comment:
Since having whitespace in the `fileNumber` is an edge case and the test here is for a happy path, I think that testing the edge case deserves its own `it` block (and `context`).